### DISCUSSION TITLE
SMTP: Enable port 25 unencrypted AUTH LOGIN

### DISCUSSION THREAD

**Discussion by stathis** [modoboa/modoboa#1777] [Q&A] [answered]
I've been trying to enable unencrypted AUTH LOGIN for one of our mailservers that run with the Modoboa package, but with no luck.

Relevant settings that had no impact:
```
@@@ /etc/postfix/master.cf

smtpd_use_tls = yes
smtpd_tls_auth_only = no
```

For IMAP & POP3, this worked just fine.
```
@@@ /etc/dovecot/conf.d/10-auth.conf

disable_plaintext_auth = no
auth_mechanisms = plain login
```

Thank you in advance :)

**Comment by tonioo** [comment]
@stathis Authentication is only available on port 587 with default install. Why do you need this?

**Comment by stathis** [comment]
We have a few legacy clients in our intranet and we need it unfortunately.

**Comment by tonioo** [comment]
@stathis Have you tried the smtpd_auth_sasl_enabled setting?

**Comment by stathis** [comment]
Yes it is on:

`smtpd_sasl_auth_enable = yes`

**Comment by tonioo** [comment]
@stathis Have you managed to fix this?

**Comment by sdeepakbz** [comment, marked as answer]
@stathis @tonioo I want to use non-TLS auth as TLS verification is adding 5-10 secs of pause when sending email over 587 TLS port. Any heads up to use non-TLS port 25?

Edit: no longer require 25 insecure port.
Fixed Pause/Delay with @MrGeneration assistance :)
https://github.com/modoboa/modoboa/issues/1977
